src/lib/client.ts

```diff
@@ -2119,6 +2119,18 @@ export default class MqttClient extends TypedEventEmitter<MqttClientEventCallbac
 		}
 	}
 
+	/**
+	 * Reschedule the ping interval regardless of reschedulePings value
+	 */
+	public reschedulePingForcely() {
+		if (
+			this.keepaliveManager &&
+			this.options.keepalive
+		) {
+			this._reschedulePing()
+		}
+	}
```

Comment:
What about adding a boolean to `reschedulePing` function like `force = false` inste4ad of creating a new method?

```ts
/**
	 * Reschedule the ping interval
	 */
	public reschedulePing(force = false) {
		if (
			this.keepaliveManager &&
			this.options.keepalive &&
			(force || this.options.reschedulePings)
		) {
			this._reschedulePing()
		}
	}

```